### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by Maxi2022gt** [Xaymar/obs-stream-effects#899] [General]
Hi, i use OBS Studio to almost all my videos.
I installed this to see if it works and it works!
The only problem is that if i put SDF Shader, and make the value low or high, my entire computer freezes.
How do i fix this?

**Comment by skeletonbow** [comment]
The SDF shader is a very GPU resource intensive filter that should only be used sparingly.  Using high settings will cause the GPU to be heavily loaded in order to perform the effect, which may result in other software such as games or OBS which use the GPU to become starved of GPU resources and stutter.  Setting the settings too high can literally use 100% of your GPU which in turn will result in system slowdowns.  Avoid using it in this manner in order to avoid this problem.

**Reply by Maxi2022gt** [reply]
Oh thanks! How do i uninstall StreamFX?

**Reply by Xaymar** [reply]
On Windows, if you installed as a static install an uninstaller will be available in Programs & Features. On every other platform, just delete the files that the unarchiver placed.

**Reply by Maxi2022gt** [reply]
I found "unins000" with the logo of StreamFX in the directory from OBS Studio, does that match?

**Reply by Xaymar** [reply]
Does it have the StreamFX logo? If not, that's not it. If you really installed StreamFX "the way its intended to be installed", you will have an uninstaller available in Programs & Features. For portable and archive releases, you will have to manually delete all StreamFX files.
